.travis.yml

```diff
@@ -11,3 +13,10 @@ install:
 
 script:
   - make flake
+  - make test
```

Comment:
They must be in separate jobs.

Reply:
I do not like this idea at all. No point bootstrap worker for 1 second flake command

Reply:
it is not necessary.

Reply:
It's a wise decision to have different things in different places with separated non-messed logs.

Reply:
I already saw this in other places. I do not endorse spaghetti.

Reply:
We have limited number of workers per organization better to do useful work instead of doing pip install for no reason

Reply:
Please let me take care of CI. I think if needed I can get them to increase those limits. But in general, they are perfectly fine. All problems exist because of features misuse, like putting tons of spaghetti in makefile instead of letting CI do only what's needed on per-job basic. Current practices of putting a ton of useless stuff in there and running all tests prevent Travis from doing better scheduling and parallelization in general.

There's a real problem with limits though: @pyup-bot generates a shitload of spam, as I explained earlier @ https://github.com/orgs/aio-libs/teams/admins/discussions/13. This is what actually needs addressing.

P.S. If you want to provide users with a good app template for reference, you must think about stuff from the perspective of the public API.

Reply:
Sure, my only requirement is minimal worker time usage so we let other projects build PRs.

Reply:
That's not the case when you abuse one job, which might've aborted earlier, with multiple tasks. So the presented here is a bad practice and I don't recommend it.